File: Ruby/fizz_buzz.rb

class FizzBuzz

	def initialize
		puts "Pick a number to know if its a Fizz or a Buzz?"
		@num = gets.chomp.to_i
	end

	def is_fizzbuzz?
	  case
	  when @num % 15 == 0 then puts "FizzBuzz"
	  when @num % 3  == 0 then puts "Fizz"
	  when @num % 5  == 0 then puts "Buzz"
	  else puts "Not a FizzBuzz"
	  end
	end
	
end


fb = FizzBuzz.new
fb.is_fizzbuzz?

File: Python/fizzbuzz.py
def fizzBuzz(self, n):
    """
    :type n: int
    :rtype: List[str]
    """
    ans = []
    fizz_buzz_dict = {3 : "Fizz", 5 : "Buzz"}

    for num in range(1,n+1):
        num_ans_str = ""
        for key in fizz_buzz_dict.keys():
            if num % key == 0:
                num_ans_str += fizz_buzz_dict[key]
        if not num_ans_str:
            num_ans_str = str(num)
        ans.append(num_ans_str)  
    return ans
